app/views/circuitverse_assets/_search_bar.html.erb

```diff
@@ -1,9 +1,23 @@
 <div class="container navbar-search-container">
   <div class="row center-row">
   <%= form_tag("/search", method: "get", id: "search-box", class: "navbar-search-bar-form") do %>
-    <%= select_tag :resource, options_for_select(%w[Users Projects], selected: params[:resource]), class: "navbar-search-bar-select", id: "search_type" %>
-    <%= text_field_tag :q, params[:q], placeholder: "Search for projects or users", autocomplete: "off", class: "form-control form-input navbar-search-bar-input" %>
+    <%= select_tag :resource, options_for_select(%w[Users Projects], selected: params[:resource]), class: "navbar-search-bar-select", id: "search_type", onchange: "changePlaceholder()" %>
+    <%= text_field_tag :q, params[:q], placeholder: "Search for projects or users", autocomplete: "off", class: "form-control form-input navbar-search-bar-input", id: "q" %>
```

Comment:
please assign a meaningful id for the element.

Reply:
Done @tanmoysrt .